helm/managed/logging/elastic/values.yaml

```diff
@@ -21,6 +22,45 @@ elasticsearch:
       # Add default labels for the volumeClaimTemplate fo the StatefulSet
       #enabled: false
     #annotations: {}
+  extraEnvs:
+    # Increase WAIT_ES_READY_IN_SEC if get 'Cluster health check is unknown' message in log
+    - name: WAIT_ES_READY_IN_SEC
+      value: "120"
+  lifecycle:
+    postStart:
+      exec:
+        command:
+          - bash
+          - -c
+          - |
+            PIPELINE_NAME=hpccpipeline
+            ES_URL=http://localhost:9200
+            ES_CLUSTER_HEALTH_PATH=_cluster/health
+            ES_CLUSTER_HEALTH_PARAMS=_cluster/health
+            PIPELINE_REQ_PATH=_ingest/pipeline
+            [ -z ${WAIT_ES_READY_IN_SEC} ] && WAIT_ES_READY_IN_SEC=100
```

Comment:
2 different default values for WAIT_ES_READY_IN_SEC? 100 here, and 120 in line 28?

Reply:
Changed to 120